ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1411,6 +1411,12 @@ class JITModBuilder {
             ResumeAfter = make_tuple(CurInstrIdx, CI);
             break;
           } else {
+            StringRef S = CF->getName();
+            if (S == "_setjmp" || S == "_longjmp") {
```

Comment:
What's with the leading `_`? Is that platform specific?

Reply:
`_setjmp` only restores registers/stack, whereas `setjmp` restores the signal mask. I was thinking whether I should add the others too, but thought I hold out until I actually see them in the wild.

Reply:
OK

Reply:
Why not add something like `assert(S != "setjmp)` ?